core/src/cluster_info.rs

```diff
@@ -1331,6 +1341,7 @@ pub struct Sockets {
     pub gossip: UdpSocket,
     pub tvu: Vec<UdpSocket>,
     pub tpu: Vec<UdpSocket>,
+    pub forwarder: Vec<UdpSocket>,
```

Comment:
Why does there need to be separate forwarder ports?

Reply:
These receive blobs.  Since these are comming from validators, they can aggregate all the transactions.

Reply:
Every name in this structure is the name of the component the sender is targeting. This "forwarder" takes the name of the component that sent it?

Reply:
The forwarder ports are also receiving the forwarded blobs from other validators; there is no "forwarder" component sending them. per se, the banking stage is responsible for forwarding unprocessed transactions.

Reply:
`tpu_via_blobs`?

Reply:
Looking forward to the day folks aim to one-up my terminology suggestion instead of just running with the first "better" thing that popped into my head.